### ISSUE TITLE
<p> tags not stripped from multi-line text field in email notifications

### ISSUE THREAD

**Issue by jmauzyk** [verbb/formie#761]
**Description**
When outputting a multi-line text field for an email notifications, the `<p>` tags don't get stripped like they seem to for other fields. As a result, the HTML ends up with nested `<p>` elements which produce some weird formatting.

**Steps to reproduce**
1. Create an email notification. In the **Email Content**, enter a heading for the multi-line field.
2. Press shift-return to enter a line break and add the multi-line text field variable.
3. Save the form and test the email notification. Output will look like

> **My Field Heading**
>
> My multi-line field content
> <br>
> **Something Else**
>>>

...instead of...

> **My Field Heading**
> My multi-line field content
>
> **Something Else**

### Form settings
- Multi-page form: Yes
- Submission Method: Page Reload
- Client-side Validation: Yes
- Custom Form Templates: No

**Additional info**
- Plugin version: 1.5.9
- Craft version: Pro 3.7.33
- Multi-site: No

**Comment by engram-design** [comment]
Right, so this is because we wrap the content in a paragraph tag as per https://github.com/verbb/formie/issues/745.

Fixed for the next release. To get this fix early, change your `verbb/formie` requirement in `composer.json` to:

```
"require": {
  "verbb/formie": "dev-craft-3 as 1.5.9",
  "...": "..."
}
```

Then run `composer update`.

**Comment by engram-design** [comment]
Fixed in [1.5.10](https://github.com/verbb/formie/releases/tag/1.5.10)
